### PR TITLE
MERL-304: Support channel documentation

### PR DESCRIPTION
**Pull request by matthewguywright**
## Description
Added a support channels document for end users.

## Documentation Changes
Added a support-channels.mdx file

### CODE REVIEW

File: internal/website/docs/support-channels.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Use one of the channels below to contact us for support.
+
+<a href="/docs/img/telemetry-cli-prompt.png" target="_blank">
+  Github
+</a> - Faust Github documentation and codebase.
+
+<a href="/docs/img/telemetry-cli-prompt.png" target="_blank">
+  Discord
+</a> - Interactive chat support on Discord.
```
Comment (josephfusco) [inline internal/website/docs/support-channels.mdx:17]:
```suggestion
<a href="https://github.com/wpengine/faustjs" target="_blank" rel="noopener noreferrer">
  GitHub
</a> - Faust Github documentation and codebase.

<a href="https://discord.gg/J2khkF9XYK" target="_blank" rel="noopener noreferrer">
  Discord
</a> - Interactive chat support on Discord.
```


- [More information on the target="_blank" vulnerability](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/)

**Review by josephfusco — CHANGES_REQUESTED** [review]

File: internal/website/docs/support-channels.mdx (hunk 1 above)
Comment (TeresaGobble) [reply internal/website/docs/support-channels.mdx:17]: Hi Matt, good work on adding the doc! It looks like you need a line break between the two links and discriptions, here, and to update the "last updated on" section:

image

I'm also not able to see the link for the doc in the sidebar- am I missing it somewhere?

image

Comment (TeresaGobble) [reply internal/website/docs/support-channels.mdx:17]: After looking at the repo again, it seems like you need to add an object in the `sidebars.js` file in `internal/website`, with a type, label, and id for it to show up in the sidebar correctly.

**Review by TeresaGobble — CHANGES_REQUESTED** [review]
Thank you for your hard work, Matt! I think this needs an object added to the sidebars.js page so folks can see your support doc there-maybe under the main header, or the subheader 'Guides.'

Looks like a line break is needed between the two links as well. Great work!

File: internal/website/docs/support-channels.mdx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+title: Support Channels
+description: Discover the Faust Support Channels
+---
+
+## Need Faust support? Look no further!
+
+Use one of the channels below to contact us for support.
+
+<a
+  href="/docs/img/telemetry-cli-prompt.png"
```
Comment (TeresaGobble) [inline internal/website/docs/support-channels.mdx:12]:
```suggestion
  href="https://github.com/wpengine/faustjs"
```

File: internal/website/docs/support-channels.mdx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  rel="noopener noreferrer"
+  target="_blank">
+  Github
+</a> - Faust Github documentation and codebase. <br />
+<a
+  href="/docs/img/telemetry-cli-prompt.png"
```
Comment (TeresaGobble) [inline internal/website/docs/support-channels.mdx:18]:
```suggestion
  href="https://discord.gg/J2khkF9XYK"
```

**Review by TeresaGobble — CHANGES_REQUESTED** [review]
Nice work, Matt!

I only have two suggestions- it looks like this branch doesn't contain recent changes that have been merged from `canary` yet today, so I think you may need to just pull those in real quick and resolve any conflicts. You can check to see if it's up to date with the `canary` branch by checking to see if today's blog post is loaded.

image

I'm also noticing that the `hrefs` to the github and discord pages here: 
image

Are still showing a redirect to some placeholders:

image

I think once you accept the href changes and update from `canary,` this one should be good to go! Great work!

File: internal/website/docs/support-channels.mdx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+## Need Faust support? Look no further!
+
+Use one of the channels below to contact us for support.
+
+<a
+  href="https://github.com/wpengine/faustjs"
+  rel="noopener noreferrer"
+  target="_blank">
+  Github
+</a> - Faust Github documentation and codebase. <br />
```
Comment (josephfusco) [inline internal/website/docs/support-channels.mdx:16]:
```suggestion
</a> - Faust GitHub documentation and codebase. <br />
```

File: internal/website/docs/support-channels.mdx [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+## Need Faust support? Look no further!
+
+Use one of the channels below to contact us for support.
+
+<a
+  href="https://github.com/wpengine/faustjs"
+  rel="noopener noreferrer"
+  target="_blank">
+  Github
```
Comment (josephfusco) [inline internal/website/docs/support-channels.mdx:15]:
```suggestion
  GitHub
```

**Review by josephfusco — CHANGES_REQUESTED** [review]
Left some additional feedback :)

**Review by TeresaGobble — APPROVED** [review]
I'm not sure what Joe's chances indicate- it looks like the same lines are suggested, but that may only be the github preview. All the changes I requested are good to go, so I'll push an approval up. Thanks Matt!

**Pull request merged**